doc/source/parameters/cfd/cahn_hilliard.rst

```diff
@@ -38,14 +38,19 @@ The default values of the Cahn-Hilliard parameters are given in the text box bel
     set potential smoothing coefficient = 1
 
     subsection epsilon
-      set method = automatic
-      set value  = 1
+      set verbosity = quiet
+      set method    = automatic
+      set value     = 1
     end
   end
   
 * ``potential smoothing coefficient``: defines the :math:`\xi` parameter in the equations above. Its value should be increased if the potential presents excessive oscillations (in advective problems for instance).
 
-* ``epsilon``: defines the :math:`\epsilon` parameter. It can either be user-defined or determined automatically for each cell. For the latter, epsilon is equal to two times the characteristic length of the cell (:math:`h`). The choices are ``automatic`` (default) or ``manual``.
+* ``epsilon``: defines the :math:`\epsilon` parameter.
```

Comment:
maybe here I would add a reference to the theory guide to recall the role of the epsilon parameter. Because otherwise this is not easy to figure out